static/js/stream_create.js

```diff
@@ -169,8 +171,23 @@ function create_stream() {
     var stream_name = $.trim($("#create_stream_name").val());
     var description = $.trim($("#create_stream_description").val());
     var is_invite_only = $('#stream_creation_form input[name=privacy]:checked').val() === "invite-only";
+    var is_invite_only_public_history = $('#stream_creation_form input[name=privacy]:checked').val() === "invite-only-public-history";
```

Comment:
The code would be cleaner and more readable if we did here:

`var private_model = $('#stream_creation_form input[name=privacy]:checked').val()`

And then had the if block below around specific values for this parameter.  

(Both here and in the next file).